src/assessments/seo/keyphraseLengthAssessment.js

```diff
@@ -1,31 +1,91 @@
-var AssessmentResult = require( "../../values/AssessmentResult.js" );
-
+let Assessment = require( "../../assessment.js" );
+let AssessmentResult = require( "../../values/AssessmentResult.js" );
+const inRange = require( "lodash/inRange" );
+const merge = require( "lodash/merge" );
 /**
- * Assesses the keyphrase presence and length
- *
- * @param {Paper} paper The paper to use for the assessment.
- * @param {Researcher} researcher The researcher used for calling research.
- * @param {Jed} i18n The object used for translations
- * @returns {AssessmentResult} The result of this assessment
-*/
-function keyphraseAssessment( paper, researcher, i18n ) {
-	var keyphraseLength = researcher.getResearch( "keyphraseLength" );
-
-	var assessmentResult = new AssessmentResult();
-
-	if ( ! paper.hasKeyword() ) {
-		assessmentResult.setScore( -999 );
-		assessmentResult.setText( i18n.dgettext( "js-text-analysis", "No focus keyword was set for this page. " +
-			"If you do not set a focus keyword, no score can be calculated." ) );
-	} else if ( keyphraseLength > 10 ) {
-		assessmentResult.setScore( 0 );
-		assessmentResult.setText( i18n.dgettext( "js-text-analysis", "The keyphrase is over 10 words, a keyphrase should be shorter." ) );
+ * Assessment to check whether the keyphrase has a good length.
+ * 1-4 words: green; 5-8 words: orange; 9+ words: red
```

Comment:
I wouldn't mention the boundary values in the description of the assessment. These are the default values, but they might differ depending on the assessor that's calling the assessment (e.g., cornerstone assessor).